pyproject.toml

```diff
@@ -21,6 +21,7 @@ exclude = [
     ".venv",
     "report",
     "docs",
+    "lib",
```

Comment:
Note: this is something we can exclude for this project, but some of our projects use locally managed source code in the lib directory. Maybe we need to use a different directory for locally managed code? So then we can keep the lib directory for charm libs.